demo-app/src/open-truss/signals.ts

```diff
@@ -0,0 +1,12 @@
+import { SignalType } from '@open-truss/open-truss'
+
+// scalar types
+export const NumbersSignal = SignalType<number[]>('number[]', [])
```

Comment:
Added a handful of basic scalar signal types. We'll need to add more here if we want to use more scalars.

Using the `NumbersSignal` as an example, components wanting to use `number[]` for a prop type will need to import this Signal and add it to their prop zod type. e.g.

```typescript
export const Props = BaseOpenTrussComponentV1PropsShape.extend({
  accountIds: NumbersSignal,
})
```

You would need to declare a corresponding signal in the config

```yaml
workflow:
  signals:
    accountIds: number[]
```